Gemfile

```diff
@@ -9,7 +9,7 @@ group :test, optional: true do
   gem 'rdoc', '~> 5.1.0'
   gem 'pry'
   gem 'addressable', '~> 2.3.8'
-  gem 'delayed_job' if ruby_version >= Gem::Version.new('2.2.2')
+  gem 'delayed_job', ruby_version < Gem::Version.new('2.5.0') ? '4.1.7': '>4.1.7' if ruby_version >= Gem::Version.new('2.2.2')
```

Comment:
The ternary combined with the one-line if statement makes this a bit harder to read. Consider
```ruby
if ruby_version >= Gem::Version.new('2.2.2')
  gem 'delayed_job', ruby_version < Gem::Version.new('2.5.0') ? '4.1.7' : '> 4.1.7'
end
```

Reply:
Agreed, I was just following precedent of the proceeding lines. I'll tidy now...